### ISSUE TITLE
Document storage

### ISSUE THREAD

**Issue by tarkal** [ff36/jnoc#70]
Access all literature, manuals, data sheets, etc. If a user has been SOLiD certified, give him/her automatic access to DAStrax where they can peruse documents, training materials, download product firmware, etc. (along with existing abilities like create ticket, access SOLiD U, etc.)

**Comment by cantorfan** [comment]
Does it mean that we need to create a file space for the dastrax and allow user to download all files on that?

**Comment by tarkal** [comment]
Yes we need to create a new section in the navigation called `Resources`. This should be an `AWS S3` backed storage facility. 

![screen shot 2015-09-15 at 10 20 52](https://cloud.githubusercontent.com/assets/7042623/9871459/11a67420-5b94-11e5-86bd-caa507dd3d3c.png)

Administrators (with appropriate permissions) should be able to perform CRUD operations on folders, and files. Ideally it should have a Drag n Drop interface for the directory structure and file upload. 

Users with appropriate permissions should be able to view / download the files.

**Comment by cantorfan** [comment]
could you please a AWS user/password and access key, so that we can continue working

**Comment by tarkal** [comment]
Do you just need access to S3?

**Comment by cantorfan** [comment]
yes, I need an env for testing purpose

2015-10-12 16:02 GMT+08:00 tarkal notifications@github.com:

> Do you just need access to S3?
>

**Comment by cantorfan** [comment]
never mind, I signed up a testing account on AWS
